pkg/virtctl/vmexport/vmexport.go

```diff
@@ -193,6 +224,9 @@ func NewVirtualMachineExportCommand(clientConfig clientcmd.ClientConfig) *cobra.
 	cmd.Flags().BoolVar(&insecure, "insecure", false, "When used with the 'download' option, specifies that the http request should be insecure.")
 	cmd.Flags().BoolVar(&keepVme, "keep-vme", false, "When used with the 'download' option, specifies that the vmexport object should not be deleted after the download finishes.")
 	cmd.Flags().StringVar(&ttl, "ttl", "", "The time after the export was created that it is eligible to be automatically deleted, defaults to 2 hours by the server side if not specified")
+	cmd.Flags().StringVar(&manifestOutputFormat, "manifest-output-format", "", "manifest output format, defaults to Yaml. valid options are yaml or json")
+	cmd.Flags().StringVar(&exportUrl, "export-url", "", "Specify url to use instead of the external URL in the export status")
```

Comment:
I think we have to be a little more specific here, it looks to me that the only thing that will currently work here is if user `kubectl port-forwards` to the export service.  Not the export proxy.

Reply:
Well essentially this is saying use this export URL to connect to. This can either be a port-forward, or a nodeport, or something else. The returned manifest will only point to the internal links. I totally agree it is not optimal, but I really don't want to do client side manipulation of the returned manifest.

Reply:
As we discussed it might be hard to figure out the right URL if we forward to the proxy service, so will just support going to the internal service.